en_US/data-integration/rule-sql-builtin-functions.md

```diff
@@ -347,18 +347,19 @@ EMQX Enterprise also supports using schema encoding and decoding functions to en
-| `unix_ts_to_rfc3339` | Convert an unix epoch to RFC3339 time string                 | 1. Unix epoch <br>2. Time unit                               |
-| `rfc3339_to_unix_ts` | Convert an RFC3339 time string (in second) to unix epoch     | 1. Time string of format RFC3339                             |
-| `rfc3339_to_unix_ts` | Convert an RFC3339 time string to unix epoch with a self-defined unit | 1. Time string of format RFC3339 <br>2. Time unit            |
-| `format_date`        | Timestamp to formatted time                                  | 1. Time unit (can be `second`, `millisecond`, `microsecond` or `nanosecond`)<br>2. Time offset (refer to [Time Offset definition](https://github.com/emqx/emqx-docs/blob/762ade37e1bfb112b30f8e202c9770936ef44623/en_US/data-integration/rule-sql-builtin-functions.md#:~:text=11%3A59%3A59-,Time%20Offset,-Offset))<br>3. Date format (refer to [Time String Codec Format](https://github.com/emqx/emqx-docs/blob/762ade37e1bfb112b30f8e202c9770936ef44623/en_US/data-integration/rule-sql-builtin-functions.md#:~:text=Syntax%20of%20Time%20String%20Format)) <br>4. Timestamp (optional parameter, default is current time) |
-| `date_to_unix_ts`    | Formatted time to timestamp                                  | 1. Time unit (can be `second`, `millisecond`, `microsecond` or `nanosecond`) <br>2. Time offset (optional, when not filled, use the time offset in the formatted time string, refer to the refer to [Time Offset definition](https://github.com/emqx/emqx-docs/blob/762ade37e1bfb112b30f8e202c9770936ef44623/en_US/data-integration/rule-sql-builtin-functions.md#:~:text=11%3A59%3A59-,Time%20Offset,-Offset) <br>3. Date format (refer to [Time String Codec Format](https://github.com/emqx/emqx-docs/blob/762ade37e1bfb112b30f8e202c9770936ef44623/en_US/data-integration/rule-sql-builtin-functions.md#:~:text=Syntax%20of%20Time%20String%20Format) <br>4. Formatted time string |
+| Function                     | Purpose                                                      | Parameters                                                   |
+| ---------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| `now_timestamp`              | Return the current unix epoch timestamp<br>Unit: `second`    | -                                                            |
+| `now_timestamp`              | Return the current unix epoch timestamp with a self-defined unit | Time unit                                                    |
+| `now_rfc3339`                | Create the current RFC3339 time string<br/>Unit: `second`    | -                                                            |
+| `now_rfc3339`                | Create the current RFC3339 time string with a self-defined unit | Time unit                                                    |
```

Comment:
Only the formatting (which is not visible in the rendered document) was changed on the line that you are commenting on. The only non-formatting change introduced by this PR is the addition of the function `timezone_to_offset_seconds` in the end of this PR.

Sorry, it is my editor that automatically change the formatting of the whole table while editing so I was actually not aware that this line had changed at all.

Your comment is probably relevant but I think it can be fixed in another PR.